components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
                     var lnk = "{{ webclient_path }}";
+                {% elif webclient_pathname %}
+                    var lnk = location.protocol + "//" + location.host + "{{ webclient_pathname }}";
                 {% else %}
                     var lnk = location.href.substring(0,location.href.length - location.search.length);
```

Comment:
I think the final ```else``` code was simply what I had originally, and maybe I was simply leaving it there in case the ```webclient_path``` didn't work in some situations. But no evidence that it was needed at all.